gnomad/utils/filtering.py

```diff
@@ -529,3 +529,57 @@ def split_vds_by_strata(
     )
 
     return [hl.vds.filter_samples(vds, list(s)) for strata, s in s_by_strata.items()]
+
+
+def remove_items_from_freq(
+    ht: hl.Table, items_to_remove: Union[Dict[str, List[Any]], List[Any]]
+) -> hl.Table:
+    """
+    Script to remove items from the freq array and freq_meta array in the Table.
+
+    :param ht: Input Table with freq and freq_meta arrays.
+    :param items_to_remove: Dictionary or list of items to remove from the freq and freq_meta arrays.
```

Comment:
What if I want to remove only frequencies with both "pop": "afr" and "sex": "XX"?

Reply:
You have to make your dictionary like this: {"pop": ["afr"]}, values to a key, even only one value has to be in a list.

Reply:
Yeah, I understand the structure of the input, but I'm asking how I can remove only frequencies with both "pop": "afr" and "sex": "XX"?

If I do: `items_to_remove={"pop": ["afr"], "sex": ["XX"]}`

It will remove any item with either, not just both

Reply:
I want an option to use the `&` operator instead of the `|` operator

Reply:
I think my original code works on this: {"pop": ["afr"], "sex": ["XX"]}
I’m having a hard time using & instead of |